src/linked_list.js

```diff
@@ -0,0 +1,155 @@
+class Node {
```

Comment:
Same notes apply for the `LinkedList` as I left for the doubly linked list, above.